ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -184,11 +176,11 @@ import Purchases
                       let transaction = transaction {
                 completion([
                     "purchaserInfo": purchaserInfo.dictionary,
-                    "productIdentifier": transaction.payment.productIdentifier
+                    "productIdentifier": transaction.sk1Transaction!.payment.productIdentifier
```

Comment:
these force-unwraps work because we're explicitly setting `useSk2IfAvailable` to `false`. We'll remove them once we update the public API, though, so they're not something to worry about.